packages/graph/clients/typescript/api.ts

```diff
@@ -2078,36 +2078,23 @@ export interface UnresolvedDecisionTimeImage {
   axis: DecisionTime;
   /**
    *
-   * @type {TimeIntervalBound}
+   * @type {UnresolvedDecisionTimeImageEnd}
    * @memberof UnresolvedDecisionTimeImage
    */
-  end?: TimeIntervalBound;
+  end: UnresolvedDecisionTimeImageEnd | null;
   /**
    *
-   * @type {TimeIntervalBound}
+   * @type {UnresolvedDecisionTimeImageEnd}
    * @memberof UnresolvedDecisionTimeImage
    */
-  start?: TimeIntervalBound;
+  start: UnresolvedDecisionTimeImageEnd | null;
```

Comment:
The property here is `start` but the type is `UnresolvedDecisionTimeImageEnd`. Is the struct itself misnamed?

Reply:
`utoipa` wrongly generates the type parameter, but it's aliased to `TimeIntervalBound`, which is correct.